src/main/java/seedu/duke/command/ViewNoteCommand.java

```diff
@@ -68,8 +73,17 @@ public String execute() {
             }
         }
         if (!noteExists) {
-            return "This note does not exists in the notebook";
+            return COMMAND_UNSUCCESSFUL_MESSAGE;
         }
-        return note.getContent();
+        ArrayList<Tag> noteTags = note.getTags();
+
+        // format output string
+        String stringToPrint = note.getTitle() + InterfaceManager.LS;
+        for (Tag tag:noteTags) {
+            stringToPrint += tag.toString() + InterfaceManager.LS;
+        }
```

Comment:
I think it would look nicer if the tags are on the same line as the title. i.e.

CS2113 Notes [CS2113] [Important] [School]

Reply:
Alright, will change that before merging